### ISSUE TITLE
Keyboard focus indicator is not visible on multiple controls present in Create new retrospective dialog, while navigating using tab key

### ISSUE THREAD

**Issue by polatengin** [microsoft/vsts-extension-retrospectives#596]
Other instances:
Other instance 1: Collect tab
path: Retrospective page--> collect tab.
Issue: Visual focus indicator is not visible when focus lands on multiple controls in the page, while navigating using tab key. It should be properly visible as a rectangular border and its luminosity should be greater than 3:1.
Controls are: MCAPS accessibility team control, testing menu control and list items present in it, collect, vote, group, act, all the tiles in feedback section, edit fields in tiles at feedback section. grouping items controls in tiles in feedback section.

Other instance 2: Group feedback dialog
Path: Retrospective application -->Groups tab --> More options (...) control in feedback section -->Group feedback control--> Group feedback dialog.
Issue: Visual focus indicator is not visible when focus lands on search results displayed on entering some data in edit field, while navigating using tab key. It should be properly visible as a rectangular border and its luminosity should be greater than 3:1.

Other instance 3: Votes tab
Path: Retrospective page-->Votes tab
Issue: Visual focus indicator is not visible when focus lands on multiple controls in the page, while navigating using tab key. It should be properly visible as a rectangular border and its luminosity should be greater than 3:1.
Controls are: all the tiles in feedback section, 4 items, Vote, Unvote, edit fields in tiles at feedback section. 
PFA (Other Instance_3_Visual focus indicator is not visible when focus lands on multiple controls in the page, while navigating using tab key.)

Other instance 4: History tab
Issue: Visual focus indicator is not visible when focus lands on expanded/collapsed controls in the page, while navigating using tab key. It should be properly visible as a rectangular border and its luminosity should be greater than 3:1.
Path: Retrospective page-->History tab.

Impact on a Person with Disability (PWD):
Users who rely on keyboard and low vision users are finding it difficult to track the tab focus as Keyboard focus indicator is not visible on multiple controls present in Create new retrospective dialog, while navigating using tab key.

Test Environment:
OS: Windows 11 [Version 23H2 (OS Build 22631.2715)]
Browser: Edge Version (119.0.2151.97), Chrome Version (119.0.6045.200)
URL: [Work items - Boards (visualstudio.com)](https://worldwidelearning.visualstudio.com/MCAPS%20Accessibility/_workitems/myactivity/)

Repro Steps:
Step 1: Open MCAPS ADO in Edge/Chrome browser.
Step 2: Navigate to 'Retrospective' in left navigation and invoke it.
Step 3: Navigate to '...' ellipsis control and invoke it.
Step 4: Navigate to Create new retrospective button and invoke it.
Step 5: Create new retrospective Dialog gets appear.
Step 6: Navigate to 'Apply template' dropdown and What went well edit field.
Step 7: Verify whether Keyboard focus indicator is visible on multiple controls present in Create new retrospective dialog, while navigating using tab key or not.

Actual Result: 
Keyboard focus indicator is not visible on multiple controls present in Create new retrospective dialog, while navigating using tab key.
Current Observation: While navigating using tab key, Keyboard focus indicator is not visible on 'Apply template' dropdown and What went well edit field.
Same issue repro for similar controls throughout the application.

Expected Result:
Keyboard focus indicator should be visible on above mentioned controls present in Create new retrospective dialog, while navigating using tab key.
Ensure that:
1. The focus indicator is properly on the controls while navigating using tab key.
2. Luminosity ratio of the focus indicator of the controls is greater than or equal to the minimum required ratio of 3:1.

Suggestive Fix:

1.Provide CSS property "outline:2px solid #000" to the element by using pesudo class ':focus'.
2.Make the same change to every element

Use CSS pseudo element selector: focus to change the styling of elements with keyboard focus.
Use focus and blur event handlers on checkboxes and radio buttons to change the styling of not only the form control, but also its label text to make it easier to see.
Styling changes should include creating a border around the interactive element and its label, typically using the CSS border or outline properties.
Provide CSS property "outline:2px solid #000" to the element by using pesudo class ': focus'.
Make the same change to every element.

MAS Reference:
[MAS 2.4.7 – Focus Visible](https://nam06.safelinks.protection.outlook.com/ap/w-59584e83/?url=https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc%3D%257B5ED5CC1A-4764-40CD-8F53-C8107BA0D5A2%257D%26file%3DMAS%25202.4.7%2520%25E2%2580%2593%2520Focus%2520Visible%2520.docx%26wdLOR%3Dc651B304E-2BC4-4A72-B13D-C09B41B14249%26action%3Ddefault%26mobileredirect%3Dtrue&data=04%7c01%7cv-rmounika%40microsoft.com%7cf318c7561aab459b6b8508d8edfafc50%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637521009518756793%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=9YLQRiJ41VkkVuN63OMIZ1O%2BaCNX%2BA6FPfp62ivjXac%3D&reserved=0)
 
WCAG Reference:
[https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html](https://nam06.safelinks.protection.outlook.com/?url=https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html&data=04%7c01%7cv-rmounika%40microsoft.com%7cf318c7561aab459b6b8508d8edfafc50%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637521009518766751%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=cJm82R74eoGkJpku/CJ4NoCMt8WepKHtsjpg8aEqpYA%3D&reserved=0)

AzDO Reference Link:
https://worldwidelearning.visualstudio.com/MCAPS%20Accessibility/_workitems/edit/1713028
